LayoutTests/accessibility/mac/html-section-elements.html

```diff
@@ -41,11 +46,35 @@
           shouldBe("body.childAtIndex(5).subrole", "'AXSubrole: AXLandmarkComplementary'");
 
           // When a header and footer are inside an article, they should not be landmarks.
+          shouldBe("body.childAtIndex(6).domIdentifier", "'article'");
+          shouldBe("body.childAtIndex(6).childAtIndex(0).domIdentifier", "'article_header'");
+          shouldBe("body.childAtIndex(6).childAtIndex(0).role", "'AXRole: AXGroup'");
           shouldBe("body.childAtIndex(6).childAtIndex(0).subrole", "'AXSubrole: '");
+          shouldBe("body.childAtIndex(6).childAtIndex(1).domIdentifier", "'article_footer'");
+          shouldBe("body.childAtIndex(6).childAtIndex(1).role", "'AXRole: AXGroup'");
+          shouldBe("body.childAtIndex(6).childAtIndex(1).subrole", "'AXSubrole: AXFooter'");
+
+	  // FIXME: How can we access <section>?
 
           // When a header and footer are inside a section, they should not be landmarks.
-          // Instead, expect only the static text inside to be exposed.
-          shouldBe("body.childAtIndex(7).role", "'AXRole: AXStaticText'");
+          shouldBe("body.childAtIndex(7).domIdentifier", "'section_header'");
+          shouldBe("body.childAtIndex(7).role", "'AXRole: AXGroup'");
+          shouldBe("body.childAtIndex(7).subrole", "'AXSubrole: '");
+
+          shouldBe("body.childAtIndex(8).domIdentifier", "'section_footer'");
+          shouldBe("body.childAtIndex(8).role", "'AXRole: AXGroup'");
+          // FIXME: AXSubrole of the footer should be empty.
```

Comment:
Curious why you say this — is it because https://w3c.github.io/html-aam/#el-footer doesn't list a subrole (just "Use WAI-ARIA mapping")? Or some other reason?

Reply:
Because the  footer element has a generic role like the header element when it is in the main element or a sectioning content element.